vue-app/src/components/WalletModal.vue

```diff
@@ -92,6 +93,7 @@ export default class WalletModal extends Vue {
     if (this.$store.state.currentRound) {
       // Load cart & contributor data for current round
       this.$store.dispatch(LOAD_USER_INFO)
+      this.$store.dispatch(LOAD_BRIGHT_ID)
```

Comment:
I wonder - should we add an `await` to `this.$store.dispatch(LOAD_USER_INFO)`?

Looks like the `LOAD_BRIGHT_ID` action depends on having `state.currentUser.walletAddress` in order to fetch the  data.

Reply:
Good catch! yes, I'll change that.